src/blocks/remote-data-no-results/render.php

```diff
@@ -7,10 +7,10 @@
 // $content (string): The block default content.
 // $block (WP_Block): The block instance.
 
-$should_render_empty_result = BlockBindings::should_render_empty_result( $block );
+$state = BlockBindings::is_error_or_empty_state( $block->context, $attributes );
```

Comment:
`$state` doesn't seem like an accurate variable name. maybe `$should_render`?

Reply:
`state` seemed the most appropriate when that method was giving back an error/empty state. After having switched it to true or false, I couldn't quite decide on what the updated name should be as technically it is driving the state of the block. That's why I left it as `state`.

Happy to change it to `should_render`, in which case the method would now be `should_render_no_result_block`